.github/workflows/integration.yml

```diff
@@ -128,6 +128,9 @@ jobs:
     - name: Run integration tests 08
       run: |
         cd test/integration/scenarios/08.webhook/ && mage test
+    - name: Run integration tests 09
+      run: |
+        cd test/integration/scenarios/09.catalogEval/ && mage test
```

Comment:
Tests number 09 has been occupied by stress test in [this commit](https://github.com/eclipse-symphony/symphony/commit/710e5d4059f73e33fc65aa5db4c7a5aeebea7d06) recently.